### ISSUE TITLE
Yarn Start: fix budget looping-test behaviour

### ISSUE THREAD

**Issue by MikeTheCanuck** [hackoregon/civic#15]
It appears that - at least for me - when the built budget package is launched, the test suite loops infinitely.  I'm seeing the same set of "0 passing", "14 passing" output over and over - no end in sight until I hit Ctrl-C.

# Repro
```
cd packages/budget
npm install
npm run build
npm run start
```

## Expected behaviour:
- the app starts and the terminal stops running commands

## Actual behaviour:
- the app starts, it can be accessed in the browser
- the terminal appears to be infinitely looping through the test suite

**Comment by DavideDaniel** [comment]
I'll fix this today if it's an issue on my branch.

**Comment by DavideDaniel** [comment]
This was already brought up, but it's a test `--watch` command intended for dev envs. I didn't see this happening in the branch where I was adding transportation so I'll assume #5 fixes it.

**Comment by MikeTheCanuck** [comment]
This behaviour appears to have been resolved at or before #37 - I no longer observe this behaviour today (and don't recall seeing it for a while now).

Comparing the "scripts" between when this was first reported and now we can confirm that [Michael's commit here](https://github.com/hackoregon/civic/commit/757ee2d9d073ea26167c2e6c03b2c40b9aeec120#diff-24c580acb15dd5ec6b67f7ab00ebe6fe) offered the relevant change.

## Relevant "scripts" entries as of Oct 22
```
    "start": "npm-run-all --parallel test:watch start:dev",
    "start:dev": "cross-env NODE_ENV=development babel-node server",
```

## Same entries as of today
```
    "start": "yarn start:dev",
    "start:dev": "cross-env NODE_ENV=development babel-node server"
```

Given we have guidance in the README for running `yarn watch` from a separate shell, I believe we have the best of both intentions.
